tools/sgen/input/FluentUIStyle.yml

```diff
@@ -148,44 +148,61 @@ Opacity:
   none: 1f
 
 AP_AvatarTokens:
-  backgroundDefaultColor: "#EDACB1"
+  backgroundDefaultColor: "FluentUIColor(light: $Colors.Neutral.white, dark: $Colors.Brand.primary)"
   borderRadius: [ xSmall: $Border.radius.none, small: $Border.radius.none, medium: $Border.radius.none, large: $Border.radius.none, xlarge: $Border.radius.none, xxlarge: $Border.radius.none ]
-  foregroundDefaultColor: "#6E0911"
+  foregroundDefaultColor: "FluentUIColor(light: $Colors.Brand.primary, dark: $Icon.accentColor)"
   presenceIconSize: [ xSmall: 0pt, small: $Icon.size.xxxSmall, medium: $Icon.size.xxxSmall, large: $Icon.size.xxSmall, xlarge: $Icon.size.xSmall, xxlarge: $Icon.size.small ]
   presenceIconOutlineColor: $Colors.Background.neutral1
   presenceIconOutlineThickness: [ xSmall: $Border.size.none, small: $Border.size.thick, medium: $Border.size.thick, large: $Border.size.thick, xlarge: $Border.size.thick, xxlarge: $Border.size.thick ]
-  ringDefaultColor: "#C50F1F"
+  ringDefaultColor: $Colors.Background.neutralDisabled
   ringGapColor: $Colors.Background.neutral1
   ringInnerGap: [ xSmall: $Border.size.thick, small: $Border.size.thick, medium: $Border.size.thick, large: $Border.size.thick, xlarge: $Border.size.thick, xxlarge: $Border.size.thicker ]
   ringThickness: [ xSmall: $Border.size.thin, small: $Border.size.thin, medium: $Border.size.thick, large: $Border.size.thick, xlarge: $Border.size.thick, xxlarge: $Border.size.thicker ]
   ringOuterGap: [ xSmall: $Border.size.thick, small: $Border.size.thick, medium: $Border.size.thick, large: $Border.size.thick, xlarge: $Border.size.thick, xxlarge: $Border.size.thicker ]
   size: [ xSmall: 16pt, small: 24pt, medium: 32pt, large: 40pt, xlarge: 52pt, xxlarge: 72pt ]
+  textCalculatedBackgroundColors: [ "NamedColor(cyanBlue10)", 
```

Comment:
these colors are specific to Avatar's only, is that right? I am not sure if I understand the significance of making them in tokens? These are just random bucket of colors used to hash out a background view?

Reply:
Correct. These are all the colors that can be used depending on the hash of the primary and secondary texts.
In the vNext tokenized world it's currently just used here, but I definitely anticipate that we'll move this to a more semantically generic place once more tokenized controls need these colors.

But the colors are not random. They are a pre-defined group of options that should be consistent. These colors are the same used by the current implementation of the AvatarView. But the design team will shortly release a new group of 30 colors to replace these. They will also include ringColors and text/foreground colors.